dev/src/reference.ts

```diff
@@ -4179,52 +4231,21 @@ export class AggregateQuery<
    * @return A Promise that will be resolved with the planner information,
    * statistics from the query execution (if any), and the query results (if any).
    */
-  explain(
+  async explain(
     options?: firestore.ExplainOptions
   ): Promise<
     ExplainResults<
       AggregateQuerySnapshot<AggregateSpecType, AppModelType, DbModelType>
     >
   > {
-    if (options === undefined) {
-      options = {};
+    const {result, explainMetrics} = await this._getResponse(
+      undefined,
+      options
```

Comment:
We need to pass an object to invoke explain.

```suggestion
      options || {}
```